single-sign-on/faqs.md

```diff
@@ -5,83 +5,215 @@ title: Single Sign-on FAQs
 
-###  Q:  How does SSO affect our automation systems and CI/CD pipelines?
+### Q: How does SSO affect our automation systems and CI/CD pipelines?
 
-Before enforcing SSO, you must create PATs for automation systems and CI/CD pipelines and use the tokens instead of a password.
+Before enforcing SSO, you must create PATs for automation systems and CI/CD pipelines and use the tokens instead of a password. When SSO is enforced, password-based authentication no longer works on your automation systems and CI/CD pipelines.
 
-When SSO is enforced, password-based authentication no longer works on your automation systems and CI/CD pipelines.
+### Q: I have a user working on projects within Docker Desktop but authenticated with personal or no email. After they purchase DB licenses, they will implement and enforce SSO via Okta to manage their users. When this user signs on SSO, is their work on DD compromised/impacted with the migration to the new account?
 
-### Q: Do I need a specific version of Docker Desktop for SSO?
+If they already have their splunk email on their account, then it will be migrated to SSO.
 
-Yes, all users in your organization must upgrade to Docker Desktop version 4.4.2 or higher. Users on older versions of Docker Desktop will not be able to sign in after enforcing SSO.
 
-### Q: Does SAML authentication require additional attributes?
+### Q: If an organization enables SSO, the owners can control DockerIDs associated with their work email domain. Some of these DockerIDs will not be users of Docker Desktop and therefore don't require a Business subscription. Can the owners choose which DockerIDs they add to their DockerOrg and get access to business features? Is there a way to flag which of these DockerIDs are Docker Desktop users?
 
-You must provide an email address as an attribute to authenticate via SAML. The ‘Name’ attribute is currently optional.
+SSO enforcement will apply to any domain email user, and automatically add that user to the Docker Hub org that enables enforcement. The admin could remove users from the org manually, but those users wouldn't be able to authenticate if SSO is enforced.
 
-### Q: When SAML SSO is enforced, at what stage is the login required to be tracked through SAML? At runtime or install time?
+### Q: Can I enable SSO and hold off on the domain verification and enforcement options?
+
+Yes, they can choose to not enforce, and people have the option to use either Docker ID (standard email/password) or email address (SSO) at the sign-in screen.Though hold off on the domain verification -- to be clear (maybe it's just the wrong language used in the question?), they'd still need to verify the domain (showing that they "own" the domain)
```

Comment:
Can we remove the internal comment about domain verification and wrong language?

Reply:
Thank you! Updated